feature-group-definitions/lab.dist.yml

```diff
@@ -0,0 +1,28 @@
+# Generated from: lab.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Lab and LCH
+description: "The CIE Lab color space expresses colors in terms of lightness and how red/green and blue/yellow a color is. LCH is a variant of Lab with polar coordinates. These color spaces can be used with the CSS `color()`, `lab()`, and `lch()` functions. Also known as CIELAB and CIELCH."
+spec: https://drafts.csswg.org/css-color-4/#cie-lab
+status:
+  baseline: low
+  baseline_low_date: 2023-05-09
+  support:
+    chrome: "111"
+    chrome_android: "111"
+    edge: "111"
+    firefox: "113"
+    firefox_android: "113"
+    safari: "15"
```

Comment:
This is earlier than the manual 15.4 version before. It matches BCD, and there's no reason to doubt the source:
https://github.com/mdn/browser-compat-data/pull/10898